sova/dashboard/routers/dependabot.py

```
@@ -0,0 +1,107 @@
+        is_dependabot_pr,
+    )
+
+    cfg = load_config(get_project_dir())
+    if not cfg.github_repo:
+        raise HTTPException(status_code=503, detail="No github_repo configured")
+
+    try:
+        all_prs = await list_open_prs(repo=cfg.github_repo, github_user=cfg.github_user)
+    except Exception:
+        log.warning("dependabot.list_prs.error", exc_info=True)
+        raise HTTPException(status_code=500, detail="Failed to fetch PRs")
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)